react/__tests__/ProductList.test.tsx

```diff
@@ -1,7 +1,8 @@
-import React, { FunctionComponent } from 'react'
+import type { FunctionComponent } from 'react'
+import React from 'react'
 import { render, fireEvent } from '@vtex/test-tools/react'
 
-import { mockItems } from '../__mocks__/mockItems'
+import { mockItems } from '../__fixtures__/mockItems'
```

Comment:
why not `fixtureItems`?

Reply:
I think I'll rename it to just `items`, it's kind of redundant to have a `fixtureItems` file inside the `__fixtures__` folder 😆

Reply:
sure, way better :feels-good: